nexus-sdk-js/src/Resource/utils.ts

```diff
@@ -33,10 +35,11 @@ export async function getSelfResource(
 export async function getResource(
   orgLabel: string,
   projectLabel: string,
-  schemaId: string,
+  schemaId: string | null,
```

Comment:
maybe it'b be nice to write a test for this?

Reply:
Maybe you're right it should be simple and not deal with this null or valued input